charts/cryosparc/values.yaml

```
@@ -0,0 +1,161 @@
+global:
+  imagePullSecret:
+    name: osc-registry
+    registry: docker-registry.osc.edu
+    username: robot$webservices-read
+    password:
+  nodeSelectorRole: webservices
+  storageClass: webservices-nfs-client
+slurmCluster: "ascend"
+license: ''
+project: ''
+workingDir: /opt/cryosparc_master
+env: []
+defaultCommand: {}
+containers: {}
+image:
+  repository: docker-registry.osc.edu/sciapps/cryosparc
+  tag:
+  pullPolicy: IfNotPresent
+
+mounts:
+  home: /users/{{ .Values.project }}/{{ include "osc.common.serviceAccountValue" . }}
+  rwDir:
+    project: /fs/ess/{{ .Values.project }}
+  roDir:
+    slurm_conf: /etc/slurm
+  socket:
+    munge_socket: /var/run/munge/munge.socket.2
```

Similar idea, get rid of the `roDir` and `socket`.

```suggestion
  slurm_conf: /etc/slurm
  munge_socket: /var/run/munge/munge.socket.2
```

If you want to keep `roDir` and `rwDir` for additional mounts, that's fine but for known mounts, don't need to use the extra key.